### ISSUE TITLE
get_data() re-downloads existing files in bigdata/ because case has been folded

### ISSUE THREAD

**Issue by hobson** [totalgood/nlpia#14]
```
>>> wv = get_data('word2vec')
100%|██████████████████████████████████████████████████████████████████████████████████| 402111/402111 [00:39<00:00, 10239.98it/s]
>>> wv = get_data('word2vec')
100%|██████████████████████████████████████████████████████████████████████████████████| 402111/402111 [00:42<00:00, 10539.42it/s]
```

**Comment by hobson** [comment]
One clue is that the file is not only lowercased but the w2v ext added seems not to be idempotent:

```
ls -al ~/code/nlpia/nlpia/src/nlpia/bigdata
total 5398592
drwxr-xr-x   6 hobsonlane  staff         192 Aug 27 09:18 ./
drwxr-xr-x  29 hobsonlane  staff         928 Aug 27 08:39 ../
-rw-r--r--   1 hobsonlane  staff  1647046227 Aug 27 09:18 googlenews-vectors-negative300.w2v.w2v.bin.gz
-rw-r--r--   1 hobsonlane  staff       31273 Aug 24 14:10 ubuntu_dialog_test.csv.gz
-rw-r--r--   1 hobsonlane  staff   552499372 Aug 24 14:42 ubuntu_dialogs.tar.gz
-rw-r--r--   1 hobsonlane  staff   552499372 Aug 24 14:45 ubuntu_dialogs.tgz
googlenews-vectors-negative300.w2v.w2v.bin.gz
```

**Comment by hobson** [comment]
Fixed in 0.2.*
